inference-engine/tests/functional/plugin/shared/include/behavior/core_threading_tests.hpp

```diff
@@ -215,6 +223,204 @@ class CoreThreadingTestsWithIterations : public ::testing::TestWithParam<CoreThr
 
     unsigned int numIterations;
     unsigned int numThreads;
+    bool isGnaTest = false;
+
+    // TODO: replace with subgraph builders after fixing *-31414
+    const std::vector<FuncTestUtils::TestModel::TestModel> defaultModels = {
+        FuncTestUtils::TestModel::convReluNormPoolFcModelFP32,
+        FuncTestUtils::TestModel::convReluNormPoolFcModelFP16
+    };
+
+    std::vector<InferenceEngine::CNNNetwork> networks;
+    void SetupNetworks(InferenceEngine::Core& ie) {
+        if (!isGnaTest) {
+            for (auto& model : defaultModels) {
+                networks.emplace_back(ie.ReadNetwork(model.model_xml_str, model.weights_blob));
```

Comment:
There are changes in [PR3331](https://github.com/openvinotoolkit/openvino/pull/3331/files) where reading IRs was removed. Why do use IRs again?

Reply:
I was not aware of this PR3331. 
The 3297 is "ready" to be merged. And it looks like 3331 is blocking it.
I am working on top of master, so I am not doing anything "again".
@dorloff @ilya-lavrenov What is the recommended order to merge these PRs?

Reply:
I see the conflicting PRs. And changes in PR3331 is more clear for me